src/all_implementations.jl

```diff
@@ -2006,6 +2010,7 @@ end
 
 is_valid(layout::UnmaskedArray) = is_valid(layout.content)
 
+Base.eltype(layout::UnmaskedArray) = eltype(layout.content)
```

Comment:
Although none of the values of an `UnmaskedArray` can actually be `missing`, this node type exists to be able to say that the data has option-type, even if it doesn't have any option-values. So, technically, this should also be a union of `Missing` and the `content` type.

```suggestion
Base.eltype(layout::UnmaskedArray) = Union{Missing, eltype(layout.content)}
```